### PR TITLE
macOS tests for NetStandard2 support

### PR DESCRIPTION
**Pull request by chamons**
- Disabled for now due to https://bugzilla.xamarin.com/show_bug.cgi?id=53164

### CODE REVIEW

File: tests/mmptest/src/NetStandardTests.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+				NetStandardTestCore (tmpDir, true);
+			});
+		}
+
+		static void NetStandardTestCore (string tmpDir, bool full)
+		{
+			TI.UnifiedTestConfig config = new TI.UnifiedTestConfig (tmpDir);
+
+			string netStandardProject = TI.GenerateNetStandardProject (config);
+			TI.RunAndAssert("/usr/local/share/dotnet/dotnet", $"restore {netStandardProject}", "Restore");
```
Comment (rolfbjarne) [inline tests/mmptest/src/NetStandardTests.cs]: Maybe check if `/usr/local/share/dotnet/dotnet` exists, and if not ignore the test? I don't have it on my system.

Comment (chamons) [reply tests/mmptest/src/NetStandardTests.cs]: Great point.

**Comment by chamons** [comment]
I disbelieve my change broke OpenTK.

```17:44:32 cp: build/tvos/reference/OpenTK-1.0.dll.config: No such file or directory
17:44:32 
make[2]: *** [build/tvos/reference/OpenTK-1.0.dll.config] Error 1
```

**Comment by chamons** [comment]
build

File: tests/mmptest/src/NetStandardTests.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Xamarin.MMP.Tests
+{
+	[TestFixture]
+	public partial class NetStandardTests
+	{
```
Comment (VincentDondain) [inline tests/mmptest/src/NetStandardTests.cs]: Super minor but it looks like you don't have the right C# formatting settings :P

**Comment by chamons** [comment]
Failure is known: https://bugzilla.xamarin.com/show_bug.cgi?id=59277

**Pull request merged**